build/optaplanner-build-parent/pom.xml

```diff
@@ -474,7 +474,6 @@
             <author>false</author>
             <failOnError>true</failOnError>
             <failOnWarnings>true</failOnWarnings>
-            <!-- Override JBoss parent otherwise it fails on JDK 16+ -->
```

Comment:
Wouldn't this override be worth trying to remove? We can track as follow-up.

Reply:
Fixed. I removed it.